wtf.go

```diff
@@ -104,51 +107,55 @@ func refreshAllWidgets() {
 	}
 }
 
+func watchForConfigChanges(app *tview.Application, configFlag *string) {
+	w := watcher.New()
```

Comment:
I prefer not to use single-character variable names except as counters in loops. Otherwise I have to keep the reference in my head when working with the code. How about `watch`?